CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 Changelog
 =========
 
+2.0.1
```

Comment:
Are you sure this is released as 2.0.1 and not as 2.1 considering it added features?